chef_master/source/server_security.rst

```diff
@@ -310,6 +310,119 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Infra Server 13.1.13:**  Chef Infra Server 13.1.13 introduces the ability to encrypt traffic between Chef Infra Server and an external PostgreSQL server over SSL.  These instructions are not all-encompassing and assume some familiarity with PostgreSQL administration, configuration, and troubleshooting. Consult the `PostgreSQL documentation <https://www.postgresql.org/docs/9.6/ssl-tcp.html>`_ for more information.
+
+The following is a typical scenario for enabling encryption between a machine running Chef Infra Server and an external machine running PostgreSQL.  Both machines must be networked together and accessible to the user.
+
+#. Run the following command on both machines to gain root access:
+
+   .. code-block:: bash
+    
+      sudo -i
+
+#. Ensure that `OpenSSL <https://www.openssl.org>`_ is installed on the PostgreSQL machine.
+
+#. Ensure that PostgreSQL is installed on the PostgreSQL machine and has SSL support compiled-in.  This applies whether you are compiling your own source or using a pre-compiled binary.
+
+#. Place SSL certificates in the proper directories on the PostgreSQL machine and ensure they have correct filenames, ownerships, and permissions.
+
+#. Enable SSL on PostgreSQL, and specify paths to the SSL certificates.  This can be done by editing ``postgresql.conf`` on the PostgreSQL machine and ensuring the relevant entries are present (substitute the appropriate paths):
+
+   .. code-block:: text
+   
+      ssl=on
+   
+      ssl_cert_file='/path/to/cert/file'
+      ssl_key_file='/path/to/key/file'
+
+#. To prevent PostgreSQL from accepting non-SSL connections, edit ``pg_hba.conf`` on the PostgreSQL machine and change the relevant Chef Infra Server connections to `hostssl`.
```

Comment:
```suggestion
#. To prevent PostgreSQL from accepting non-SSL connections, edit ``pg_hba.conf`` on the PostgreSQL machine and change the relevant Chef Infra Server connections to ``hostssl``.
```